features/step_definitions/bashcov_steps.rb

```diff
@@ -0,0 +1,96 @@
+# frozen_string_literal: true
+
+require "aruba/api"
+require "json"
+require "simplecov"
+
+# Convenience methods for use in step definitions
+module StepHelpers
+  include Aruba::Api
+
+  def aruba_setting(setting_name)
+    Aruba.configure { |config| return config.public_send(setting_name) }
+  end
+
+  def aruba_working_directory_expanded
+    File.expand_path(aruba_setting(:working_directory), aruba_setting(:root_directory))
+  end
+
+  def simplecov_results_json
+    run_simple(<<-'COMMAND', fail_on_error: false)
```

Comment:
BTW do we actually need the `fail_on_error: false` here?

Reply:
Hm.  I remember that there was a good reason for passing that option at some point, but I've removed it and the features work correctly so I guess that reason no longer obtains.